rice-middleware/impl/src/main/java/edu/arizona/rice/kim/service/impl/UaUiDocumentServiceImpl.java

```diff
@@ -3,7 +3,7 @@
  * Copyright 2009 State of Arizona Board of Regents
  *
  */
-package edu.arizona.kim.service.impl;
+package edu.arizona.rice.kim.service;
```

Comment:
rice-2.1-ua-development has the same setting in KimWebSpringBeans.xml which reference to . org.kuali.rice.kim.service.impl.UiDocumentServiceImpl. UaUiDocumentService has no definition in any spring bean config file. It's defined by not used in RT. I doubt it might be legacy code without cleanup.